File: Lession 1/LinearSearch.py
def LinearSearch(arr, x):
    for i in range(len(arr)):
        if arr[i] == x:
            return i
    return -1
def main():
    arr = [12, 11, 13, 5, 6]
    x = 13
    print(LinearSearch(arr, x))
if __name__ == "__main__":
    main()

File: Lession 1/LinearSearch.jl
function LinearSearch(arr, x)
    for i in 1:length(arr)
        if arr[i] == x
            return i
        end
    end
    return -1
end
function main()
    arr = [1, 2, 3, 4, 5]
    x = 3
    println(LinearSearch(arr, x))
end
main()